.gitignore

```diff
@@ -7,4 +7,7 @@ node_modules/
 npm-debug.log
 jsconfig.json
 dump.rdb
-.vscode
\ No newline at end of file
+.vscode
+.idea
+yarn.lock
+example/yarn.lock
```

Comment:
👍

Reply:
Though I think we should use yarn. I did use npm but to update that package-lock.json it took me 20 seconds more